components/helpers/json-to-csv.js

```diff
@@ -0,0 +1,63 @@
+define('json-to-csv', [], () =>
+    /**
+    * Converts a JSON schema+data into valid CSV string
+    * @param {object} schema - json schema used to render column headers
+    * @param {obejct} data - data array contining items to render as rows
+    */
+    function jsonToCSV(schema, data = [], separator = ' ,', lineTerminator = '\n') {
+        // eslint-disable-next-line no-param-reassign
+        schema = ((schema && schema.properties) ? schema.properties : schema || {});
+
+        // TECH-DEBT - sort out this throwing..
+        // eslint-disable-next-line no-throw-literal
+        if (separator.indexOf('"') > -1) throw 'Invalid separator';
+        // eslint-disable-next-line no-throw-literal
+        if (lineTerminator.indexOf('"') > -1) throw 'Invalid lineTerminator';
+
+        const keys = [];
+        let line = [];
+        const rows = [];
+        const headers = [];
+
+        // wraps any value containing separators in quotes
+        // need to also escape double quotes
+        const escapeValue = function (value) {
```

Comment:
Does the linter not complain about this anonymous function? Can we have either:
function escapeValue(value) {
or
const escapeValue = value => {
instead?